priv/test_files/unused_callbacks/clean.erl

```diff
@@ -0,0 +1,12 @@
+-module(clean).
+
+-callback used_callback(map()) -> boolean().
+-callback used_atom(atom()) -> list().
+
+-export([used_callback/1, used_atom/1]).
+
+used_callback(#{module := Module, state := State}) ->
+    Module:used_callback(State).
+
+used_atom(Module) ->
+    other_module:call(used_atom, Module).
```

Comment:
To make it _clearer_… let's use…
```suggestion
    erlang:apply(Module, used_atom, [another_atom]).
```
Since that's a perfectly valid way of using the callback that _is_ captured by Hank.